### PR TITLE
Merge `release-2021.11.10` into main

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

**Comment by jsignell** [comment]
See https://github.com/saturncloud/recipes/pull/6

File: examples/rstudio/.saturn/saturn.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "example-rstudio",
+    "image_uri": "saturncloud/saturn-rstudio:2021.10.26",
```
Comment (forana) [inline examples/rstudio/.saturn/saturn.json:3]: Not an issue now, but this should get bumped to `2021.11.0`. Want me to PR?

**Review by forana — APPROVED** [review]

File: examples/rstudio/.saturn/saturn.json (hunk 1 above)
Comment (jsignell) [reply examples/rstudio/.saturn/saturn.json:3]: It should get that bump on the release branch though right?

Comment (forana) [reply examples/rstudio/.saturn/saturn.json:3]: Oh, right

**Pull request merged**